specification/v1/bindings/restful/swagger.yaml

```diff
@@ -121,15 +121,15 @@ paths:
           description: Authorization identifier to evaluate.
           schema:
             type: string
-        - name: egf_id
+        - name: authority_id
           in: query
           required: true
           description: Unique identifier of the governance framework.
           schema:
             type: string
         - name: all
           in: query
-          required: true
+          required: false
```

Comment:
Can this actually be omitted? If so, what is the default? Is there such a thing as a default ecosystem governing authority associated with a TR? If so, how does this work with multiplicity?

Reply:
This is about returning ALL the authorizations back. I think we likely can and should remove this.